modules/ossm-accessing-bookinfo-application-using-istio-gateway-injection.adoc

```diff
@@ -0,0 +1,104 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-accessing-bookinfo-application-using-istio-gateway-injection"]
+= Accessing the Bookinfo application by using Istio gateway injection
+:context: ossm-accessing-bookinfo-application-using-istio-gateway-injection
+
+Gateway injection uses the same mechanisms as {istio} sidecar injection to create a gateway from a `Deployment` resource that is paired with a `Service` resource. The `Service` resource can be made accessible from outside an {ocp-product-title} cluster.
+
+.Prerequisties
+
+* You are logged in to the {ocp-product-title} web console as `cluster-admin`.
+
+* The {SMProductName} Operator must be installed.
+
+* The {istio} resource must be deployed. 
+
+.Procedure
+
+. Create the `istio-ingressgateway` deployment and service by running the following command at the CLI:
++
+[source,terminal]
+----
+$ oc apply -n bookinfo -f ingress-gateway.yaml
+----
++
+[NOTE]
+====
+This example uses a sample `ingress-gateway.yaml` https://github.com/istio-ecosystem/sail-operator/blob/eb874d1605c0401887c30d96312c6db1006d89dd/chart/samples/ingress-gateway.yaml[file] that is available in the Istio community repository. 
```

Comment:
does the url need to be that specific? would the following do instead?

https://raw.githubusercontent.com/istio-ecosystem/sail-operator/main/chart/samples/ingress-gateway.yaml

Reply:
done